action-server/covidflow/actions/daily_ci_keep_or_cancel_form.py

```diff
@@ -78,14 +79,27 @@ def request_next_slot(
         )
 
     def _utter_ask_slot_template(self, slot: str, tracker: Tracker) -> Optional[str]:
-        if slot == CANCEL_CI_SLOT:
+        error_suffix = "_error" if tracker.get_slot(REQUESTED_SLOT) == slot else ""
+        if slot == CONTINUE_CI_SLOT:
             if tracker.get_slot(SYMPTOMS_SLOT) == Symptoms.NONE:
-                return f"utter_ask_daily_ci__keep_or_cancel__{slot}_no_symptoms"
+                return f"utter_ask_daily_ci__keep_or_cancel__{slot}_no_symptoms{error_suffix}"
```

Comment:
Oh, from a readability and maintainability standpoint, I am really not sure about this...